Utilities/StaticAnalyzers/src/ArgSizeChecker.h

```diff
@@ -4,13 +4,16 @@
 #include <clang/StaticAnalyzer/Core/Checker.h>
 #include <clang/StaticAnalyzer/Core/PathSensitive/CheckerContext.h>
 #include <clang/StaticAnalyzer/Core/BugReporter/BugType.h>
+#include "FWCore/Utilities/interface/thread_safety_macros.h"
```

Comment:
This `#include` would require
```xml
<use name="FWCore/Utilities" source_only="1"/>
```
in `BuildFile.xml`.

Reply:
That header is used in other checker headers. Never knew it had to be declared.